modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/custom_range_facet.js

```diff
@@ -277,7 +277,11 @@ AUI.add(
 					instance.searchCustomRangeToggleName
 				);
 
-				if (!searchCustomRangeToggle?.hasAttribute('data-term-id')) {
+				if (
+					!searchCustomRangeToggle?.hasAttribute('data-term-id') ||
+					searchCustomRangeToggle?.getAttribute('data-selection') ===
```

Comment:
Instead of this `data-selection` attribute, can we do `searchCustomRangeToggle?.type === 'radio'`?

Reply:
Good idea! I'll add it in and take out `data-selection`